src/hooks/usePageTitle/usePageTitle.ts

```diff
@@ -0,0 +1,17 @@
+import { useEffect } from 'react';
+import { useIntl } from 'react-intl';
+
+export const usePageTitle = () => {
+  const { formatMessage } = useIntl();
+
+  useEffect(() => {
+    const translatedPageTitle = formatMessage({ id: 'header.title' });
+
+    if (document.title !== translatedPageTitle) {
+      document.title = translatedPageTitle;
+      document
+        .querySelector('meta[name="description"]')
+        ?.setAttribute('content', formatMessage({ id: 'page.description' }));
+    }
+  }, []);
```

Comment:
A to się będzie zmieniać, kiedy sobie zmienię język na dropdownie?

Reply:
Tak, zmienia się.